### ISSUE TITLE
Refactor law creation

### ISSUE THREAD

**Issue by cjerdonek** [teorth/QED#15]
I was thinking about how to simplify `main.js` further, and the next thing that occurred to me (using the criterion of easiest, biggest win) is to move the base enumeration of laws to `index.html`. For the below to work as I'm describing, can all of the `Law` objects be instantiated before the first `Exercise` objects are created?

If so, a section could be added to the data container element as follows:

```html
<div id="dataContainer">
<div id="lawsContainer">
<div class="law" id="LawConjunction1">
    <A HREF="https://en.wikipedia.org/wiki/Conjunction_introduction" target="_blank">CONJUNCTION INTRODUCTION</A>
</div>
...
</div> <!-- End lawsContainer -->
...
</div> <!-- End dataContainer -->
```

Then, prior to creating the `Exercise` objects, `main.js` could have a routine that iterates over the law elements in the HTML and instantiates a `Law` object for each one.

After that, the JS could loop over an array of triples like the following to set the `givens` and `conclusion` on each law:

```javascript
[
    ["LawConjunction1", [A, B], AND(A,B)],
    ...,
]
```

What's nice about this approach is that the HTML declaratively does the bulk of the work (which means less code, which is a good thing), and the JS is minimally responsible for only the part that _has_ to be done in the JS (because of how the givens and conclusion are defined using JS functions).

If this works, something similar can be done for the sections and `Exercise` objects within. (Currently, the reverse is happening for the `Exercise` objects, with the JS procedurally enumerating the `Exercise` objects, and the HTML decorating those objects with the additional info. But the idea would be to reverse those roles along the lines described above for the `Law` objects.)

**Comment by teorth** [comment]
I think I managed to implement this.

**Comment by cjerdonek** [comment]
I fixed this: ed14a44fa21d46e6c3d0e8810357452eb433f906. It was an extra closing div causing the data container to terminate early I think. The validator was helpful in finding the issue.
